frontend/src/routes/guide/GuidePage.vue

```diff
@@ -65,10 +65,14 @@ const breadcrumbs = `${electionName} — ${districtNameWithCode}`;
 const text =
   route.params.election === 'senatni-2022'
     ? `
-Čeká vás 30-40 otázek. Na stejné otázky nám odpověděly kandidující strany. Zodpovězení otázek zabere cca 10 minut. Na konci se dozvíte, jak se kandidáti shodují s vašimi názory.
+    Vítejte ve Volební kalkulačce pro volby do Senátu ČR.
```

Comment:
Tendle text by se mel nacitat uz z datovych souboru, aby v tom kodu nebyly ty ify - https://github.com/KohoVolit/volebni-kalkulacka-2022/blob/main/backend/converter.py#L584

Ale je to jedno. Ted tady klidne nechme ty ify a pak to muzeme presunout do datovych souboru a nacitat tady.

Reply:
Ano, to je pravda, dočasné řešení